calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| Threshold breach symptoms        | ??                                                           |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Felix open FDS
+
+| Felix open FDS                   |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum by (pod) (process_open_fds{pod=~"calico-node.*"})*      |
+| Explanation                      | Total number of opened file descriptors per calico-node pod. |
+| Threshold value recommendation   | ??                                                           |
+| Threshold breach symptoms        | ??                                                           |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority Level                   | Optional.                                                    |
+
+### Felix max FDS
+
+| Felix max FDS                    |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum by (pod) (process_max_fds{pod=~"calico-node.*"})*       |
+| Explanation                      | Maximum number of opened file descriptors allowed per calico-node pod. |
+| Threshold value recommendation   | ??                                                           |
+| Threshold breach symptoms        | ??                                                           |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority level                   | Optional.                                                    |
+
+### Felix resync started
+
+| Felix resync started             |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum(rate(felix_resyncs_started[5m]))*                       |
+| Explanation                      | Number of times that Typha has reported to Felix that it is connecting with the data store. |
+| Threshold value recommendation   | Occasional resyncs are normal.  Investigate resync counters that rapidly rise. |
+| Threshold breach symptoms        | Typha pods may be having issues or experiencing resource contention.  Some calico-nodes that are paired with Typha pods experiencing issues will not be able to sync with the data store. |
+| Threshold breach recommendations | Redeploy Typha, and check resource contention and network connectivity from Typha to the data store. |
```

Comment:
I'd put emphasis on investigating the root cause.  Redeploying Typha in a large cluster can be extremely disruptive and it could make problem worse if typha is working fine but API server is overloaded(!)

Reply:
Reworded.